lldb/include/lldb/API/SBDebugger.h

```diff
@@ -13,6 +13,7 @@
 
 #include "lldb/API/SBDefines.h"
 #include "lldb/API/SBPlatform.h"
+#include "lldb/API/SBStructuredData.h"
```

Comment:
This isn't needed in the header file right? Move to SBDebugger.cpp if it is still needed? No mention of SBStructuredData in SBDebugger.h

Reply:
Correct, I left this in here by mistake.